Std/Data/Array/Lemmas.lean

```diff
@@ -524,6 +517,13 @@ theorem mapIdx_spec (as : Array α) (f : Fin as.size → α → β)
   unfold modify modifyM Id.run
   split <;> simp
 
+theorem get_modify {arr : Array α} {x i} (h : i < arr.size) :
```

Comment:
what's this move about?

Reply:
This lemma needs `size_modify`, which used to be in `Init` but now is in this file, so I've moved `get_modify` immediately below `size_modify` (in the `modify` section).